migration/20180101-materialized-work-for-lanes.sql

```diff
@@ -5,8 +5,7 @@
 update lanes set inherit_parent_restrictions=true where _list_datasource_id is null and id not in (select lane_id from lanes_customlists);
 ALTER TABLE lanes ALTER COLUMN inherit_parent_restrictions SET DEFAULT true;
 
-drop materialized view mv_works_editions_workgenres_datasources_identifiers;
```

Comment:
Beyond this point the file used to be a copy of `core/files/materialized_view_works_workgenres.sql`, because we were modifying an existing materialized view. Now it's a copy of `core/files/materialized_view_for_lanes.sql` because we're creating a new materialized view.